### ISSUE TITLE
"Download"-ing proven theories as 'proof files' should be explained more prominently

### ISSUE THREAD

**Issue by rsasse** [tamarin-prover/manual#42]
Currently, the "Download" button is explained in the middle of a paragraph of the initial example, and this is the only place it is mentioned. 

Being able to download, and replay, proofs is an important feature though that deserves better explanation. Probably should additionally get  its own heading to help readers find it: "Storing proofs", "Storing (partial) proofs", "Storing (partial) proofs and reloading" or something like that, and then a bit more about the reloading process.

**Comment by cascremers** [comment]
Good point. Perhaps we also need a chapter with "typical workflows" for more involved protocols. This could then also point out more clearly that one needs to observe partial deconstructions, which users often miss initially.

**Comment by thomwiggers** [comment]
This could be very helpful to anyone trying to pick up and play with a larger proof such as the TLS 1.3 model.
